doc/command-line.md

```diff
@@ -993,6 +993,39 @@ size: 5177
 signature license: CC0
 ```
 
+### `sourmash signature fileinfo` - display a summary of the contents of a sourmash collection
+
+Display signature file, database, or collection.
+
+For example,
+```
+sourmash sig fileinfo tests/test-data/prot/all.zip
+```
+will display:
+```
+path filetype: ZipFileLinearIndex
+location: /Users/t/dev/sourmash/tests/test-data/prot/all.zip
+is database? yes
+has manifest? yes
+is nonempty? yes
+num signatures: 8
+** examining manifest...
+31758 total hashes
+summary of sketches:
+   2 sketches with dayhoff, k=19, scaled=100, abund
+   2 sketches with hp, k=19, scaled=100, abund
+   2 sketches with protein, k=19, scaled=100, abund
+   2 sketches with DNA, k=31, scaled=1000, abund
```

Comment:
i think you now also provide total # hashes for each sketch type?

Reply:
yep! good catch. fixed in d4facce0